html/modules/custom/reliefweb_utility/src/Helpers/LocalizationHelper.php

```diff
@@ -113,7 +113,6 @@ public static function collatedKsort(array &$items, $language = NULL) {
         $items = $reordered;
         return TRUE;
       }
-      return FALSE;
```

Comment:
This changes the logic and makes it less consistent with the other methods that return FALSE when collated sorting fails.

Defaulting to ksort when the collated sorting of the keys fails may not be appropriate. I think it's better to restore the `return FALSE`.